testcases/open_posix_testsuite/conformance/interfaces/feof/1-1.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is licensed under the GPL license.  For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *
+ *	The feof function shall test the end-of-file indicator for
+ *	the stream pointed to by argument and the function shall return
+ *	non-zero if and only if the end-of-file indicator is set for stream
+ *
+ * method:
+ *	-open file in write mode
+ *	-write 2 sample strings into the file
+ *	-close the file
+ *	-open the same file in read mode
+ *	-read first string from file and check feof return value
+ *	-read second string from file and check feof return value
+ */
+
+#define _XOPEN_SOURCE 700
```

Comment:
Please remove this #define. I will be standardizing on a newer version of the POSIX spec(s) soon: 
https://github.com/linux-test-project/ltp/pull/443 .

Reply:
Removed definition from both files.